docs/compilers/Error Log Format.md

```diff
@@ -105,11 +107,13 @@ The `rules` section contains a rule entry that corresponds to metadata or `Diagn
 4. `defaultConfiguration`: Default severity level for the diagnostics reported for the descriptor, such as `error`, `warning`, `note`, etc.
 5. `helpUri`: Help uri for help information associated with the descriptor.
 6. `properties`: One or more custom properties associated with the descriptor. It includes the following:
-  1. `category`: `Category` associated with the descriptor, such as, `Design`, `Performance`, `Security`, etc.
-  2. `isEverSuppressed` and `suppressionKinds`: If a rule had either a source suppression or was disabled for part or whole of the compilation via options, the rule metadata contains a special flag `isEverSuppressed = true` and an array `suppressionKinds` with either or both of the below suppression kinds:
-    1. `inSource` suppression kind for one or more reported diagnostic(s) that were suppressed through pragma directive, SuppressMessageAttribute or a DiagnosticSuppressor.
-    2. `external` suppression kind for diagnostic ID that is disabled either for the entire compilation (via global options such as /nowarn, ruleset, globalconfig, etc.) or for certain files or folders in the compilation (via editorconfig options).
-  3. `tags`: An array of one of more `CustomTags` associated with the descriptor.
+   1. `category`: `Category` associated with the descriptor, such as, `Design`, `Performance`, `Security`, etc.
+   2. `executionTimeInSeconds`: Execution time in seconds associated with the analyzer, as reported with the `/reportAnalyzer` compiler switch. Values less than `0.001` seconds are reported as `<0.001`.
+   3. `executionTimeInPercentage`: Execution time in percentage associated with the analyzer, as reported with the `/reportAnalyzer` compiler switch. Values less than `1` are reported as `<1`.
```

Comment:
Other two new properties being added. Again alternate property name suggestions are welcome.

Reply:
@mavasani 
For ExecutionTimeInSeconds/Percentage, I think you could save a lot of headache for our json parsing brethren by rounding the value rather than putting in "<1" or "<0.001". Then they can consume the values as doubles rather than strings.

Reply:
@bichuga In that case most of the values will be 0s, which doesn’t seem right and users might conclude analyzer never executed. I tried to match the exact value output in reportanalyzer